code/modules/integrated_electronics/core/printer.dm

```diff
@@ -29,7 +29,7 @@
 
 /obj/item/device/integrated_circuit_printer/attack_robot(mob/user as mob)
 	if(Adjacent(user))
-		return interact(user)
```

Comment:
the argument for them not being meant to use it falls sorta flat if this is already here
what other interact is there that isn't the interface?